presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst

```diff
@@ -184,3 +184,170 @@ for cleanup. Only applicable when ``enable-old-task-cleanup`` is ``true``.
+If it is ``0``, then there is no limit.
+
+``native_debug_validate_output_from_operators``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+If set to ``true``, then during execution of tasks, the output vectors of every operator are validated for consistency.
+This is an expensive check so should only be used for debugging.
+It can help debug issues where malformed vector cause failures or crashes by helping identify which operator is generating them.
+
+``native_join_spill_enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Native Execution only. Enable join spilling on native engine.
+
+``native_join_spill_memory_threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``0``
+
+Native Execution only. The max memory that hash join can use before spilling.
+If it is ``0``, then there is no limit.
+
+``native_join_spiller_partition_bits``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``2``
+
+Native Execution only. The number of bits (N) used to calculate the spilling partition number for hash join and RowNumber: ``2`` ^ N.
+
+``native_max_spill_file_size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``0``
+
+The max allowed spill file size. If it is ``0``, then there is no limit.
+
+``native_max_spill_level``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``4``
+
+Native Execution only. The maximum allowed spilling level for hash join build.
+0 is the initial spilling level, -1 means unlimited.
+
+``native_order_by_spill_memory_threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``0``
+
+Native Execution only. The max memory that order by can use before spilling.
+If it is ``0``, then there is no limit.
+
+``native_row_number_spill_enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Native Execution only. Enable row number spilling on native engine.
+
+``native_simplified_expression_evaluation_enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Native Execution only. Enable simplified path in expression evaluation.
+
+``native_spill_compression_codec``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``varchar``
+* **Default value:** ``none``
+
+Native Execution only. The compression algorithm type to compress the spilled data.
+Supported compression codecs are: ZLIB, SNAPPY, LZO, ZSTD, LZ4 and GZIP. ``none`` means no compression.
+
+``native_spill_file_create_config``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``varchar``
+* **Default value:** ``""``
+
+Native Execution only. Config used to create spill files.
+This config is provided to underlying file system and the config is free form.
```

Comment:
Is "config" right here? I'd expect configuration variable, or something like that. I think of config as all the properties.

Reply:
updated and corrected